src/renderer/account/components/TransactionsPanel/Send/Send.js

```diff
@@ -91,8 +91,13 @@ export default class Send extends React.PureComponent {
     const { confirm, receiver } = this.props;
     const symbol = this.getSymbol();
 
-    confirm(`Would you like to transfer ${this.getAmount()} ${symbol} to ${receiver}?`, {
-      title: 'Confirm fund transfer',
+    confirm((
+      <span>
+        Would you like to transfer {this.getAmount()} {symbol} to address{' '}
```

Comment:
does a <br /> not work?

Reply:
I wanted to let it flow as a sentence since different screens might render differently, especially if zooming is different.